### PR TITLE
PR: Directors films link

### PR DESCRIPTION
**Pull request by tjroeder**
User Story 10:
- Created navbar link on directors/:director_id show page to have link to directors films.
- Created feature testing for the directors films link.

### CODE REVIEW

File: spec/features/directors/show_spec.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@
       it 'should visit the page at /director/:id' do
         visit "/directors/#{director_1.id}"
 
+        save_and_open_page
```
Comment (jbreit88) [inline spec/features/directors/show_spec.rb]: Should probably take the `save_and_open_page` out before merging.

File: app/views/directors/show.html.erb [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <nav>
   <a href="/films">Films Index</a>
   <a href="/directors">Directors Index</a>
+  <a href="/directors/<%= @director.id %>/films"><%= @director.name %>'s Films Index</a>
 </nav>
 
-<h1><%= @director.name %></h1>
+<h1><%= @director.name %>'s Films Index</h1>
```
Comment (jbreit88) [inline app/views/directors/show.html.erb:7]: I like this idea.

**Review by jbreit88 — CHANGES_REQUESTED** [review]
Check the spec file and remove the launchy command.

**Review by tjroeder — COMMENTED** [review]
Removed launchy save_and_open_page method.

File: spec/features/directors/show_spec.rb (hunk 1 above)
Comment (tjroeder) [reply spec/features/directors/show_spec.rb]: Resolved.

**Review by jbreit88 — APPROVED** [review]

**Pull request merged**